lib/react_client/react_interop.dart

```diff
@@ -172,6 +170,18 @@ class InteropContextValue {
   external factory InteropContextValue();
 }
 
+/// A JavaScript interop class representing the return value of `createContext`.
+///
+/// Used for accessing Dart [Context.Provider] & [Context.Consumer] components.
+///
+/// __For internal/advanced use only.__
+@JS()
+@anonymous
+class ReactContext {
+  external get Provider;
+  external get Consumer;
```

Comment:
Do we not know what either of these types are?

Reply:
added